packages/gluestick/src/config/__tests__/compileWebpackConfig.test.js

```diff
@@ -1,8 +1,13 @@
-/* @flow */
```

Comment:
Was Flow intentionally removed? Probably we should re-add it, remove the `FlowIgnore` and reuse config files so flow is ok.